samples/main.cpp

```diff
@@ -163,8 +163,7 @@ int main(int /*argc*/, char* /*argv*/[])
     rotation += rotationAmount * delta.getSecondsFloat();
     novelChanRect->transform().rotation() = rotation;
 
-    if (rotation > 360.0f)
-    {
+    if (rotation > 360.0f) {
```

Comment:
shouldn't this be `>= 360`, since `360 == 0`?

Reply:
No idea, I just fixed the bracing.